frontend/src/pages/playback/music/index.vue

```diff
@@ -154,11 +174,30 @@ function onSlideChange(): void {
 }
 </script>
 
+<script lang="ts">
+export default {
+  data(): object {
+    return {
+      isVisualising: false
+    };
+  }
+};
+</script>
```

Comment:
Why???

Reply:
I didn't know that variable refs could be defined like this `const isVisualizing = ref(false);` until I looked at the composition API docs.